### PR TITLE
[12.0][MIG] website_google_tag_manager: Migration to v12

### PR DESCRIPTION
**Pull request by cristinamartinrod**
cc @Tecnativa

### CODE REVIEW

File: website_google_tag_manager/views/website_views.xml [hunk 1]
```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pedrobaeza) [inline website_google_tag_manager/views/website_views.xml]: Why removing this?

File: website_google_tag_manager/__manifest__.py [hunk 2]
```diff
@@ -16,10 +16,9 @@
     ],
     "data": [
         "views/res_config_settings_view.xml",
-        "views/website_views.xml",
         'views/website_templates.xml',
     ],
     'images': [
-        'static/description/icon.png',
```
Comment (pedrobaeza) [inline website_google_tag_manager/__manifest__.py]: It's OK to remove the icon here, but don't remove the file per se, as it's the module icon (it's always automatically got from that path without being here declared).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remove again `.DS_Store` file

File: website_google_tag_manager/views/website_views.xml (hunk 1 above)
Comment (cristinamartinrod) [reply website_google_tag_manager/views/website_views.xml]: En v12 Odoo remove all atributes (analytics, google maps, etc...) in this form. Now this form is just a popup to create a new website.

Comment (pedrobaeza) [reply website_google_tag_manager/views/website_views.xml]: But then you need other form where putting this. There should be a `website` form...

File: website_google_tag_manager/__manifest__.py (hunk 2 above)
Comment (cristinamartinrod) [reply website_google_tag_manager/__manifest__.py]: I don't remove it. Only was changed for new one :)

File: website_google_tag_manager/views/website_views.xml (hunk 1 above)
Comment (cristinamartinrod) [reply website_google_tag_manager/views/website_views.xml]: Doesn't look like it

File: website_google_tag_manager/__manifest__.py (hunk 2 above)
Comment (pedrobaeza) [reply website_google_tag_manager/__manifest__.py]: This is saying the contrary:

https://github.com/OCA/website/pull/552/commits/4bad4da977df42c045b41012435226aa74012a32#diff-d6dd36ca45092f55ef21b2c115018a42

As said, there MUST BE a `static/description/icon.png` file in the module for having a module icon. If you add more images (which this section is for), great, but the other one is different.

Comment (cristinamartinrod) [reply website_google_tag_manager/__manifest__.py]: Of course and here is the icon file in his branch: https://github.com/Tecnativa/website/tree/12.0-mig-website_google_tag_manager/website_google_tag_manager/static/description

Comment (pedrobaeza) [reply website_google_tag_manager/__manifest__.py]: It's very weird, as GitHub is saying that is removed:

seleccion_004

but indeed it's OK.

**Review by pedrobaeza — APPROVED** [review]

File: website_google_tag_manager/__manifest__.py (hunk 2 above)
Comment (cristinamartinrod) [reply website_google_tag_manager/__manifest__.py]: Yeah, is about the image size, now is smaller. I hadn't noticed it before either.

**Review by chienandalu — COMMENTED** [review]
I couldn't make it work

image

**Comment by cristinamartinrod** [comment]
@chienandalu this is the test on runbot with a real Container ID.
GTM approves the script and collects data:
captura de pantalla 2018-12-20 a las 12 06 38
captura de pantalla 2018-12-20 a las 12 06 30

**Review by chienandalu — APPROVED** [review]
Ok, then I just had it misconfigured it :+1:

**Pull request merged**